resources/languages/de.json

```diff
@@ -806,5 +811,6 @@
 	"SYSINFO_SD_CID": "SD CID: %s\n",
 	"SYSINFO_SYSTEM_ID0": "System ID0: %s\n",
 	"SYSINFO_SYSTEM_ID1": "System ID1: %s\n",
-	"SORTING_TICKETS_PLEASE_WAIT": "Tickets werden sortiert, bitte warten..."
+	"SORTING_TICKETS_PLEASE_WAIT": "Tickets werden sortiert, bitte warten...",
+	"LUA_NOT_INCLUDED": "This build of GodMode9 was\ncompiled without Lua support."
```

Comment:
```suggestion
	"LUA_NOT_INCLUDED": "GodMode9 wurde ohne\nUnterstützung für Lua  gebaut."
```

Reply:
Unsure if GM9 supports Umlauts. If it does, it's fine.